clearpath_socketcan_interface/package.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>clearpath_socketcan_interface</name>
+  <version>0.8.2</version>
```

Comment:
This needs to be changed to match the rest of the packages.

Reply:
I changed it but what about the changelog file?

Reply:
Just just get rid of the changelog.